src/components/summary/_macro.njk

```diff
@@ -104,7 +104,10 @@
                                             </dd>
                                         {% endif %}
                                         {% if item.actions %}
-                                            <dd class="ons-summary__actions{{ ' ons-summary__column-size--2' if not item.valueList }}" {% if item.id %}id="{{ item.id }}"{% endif %}>
+                                            <dd
```

Comment:
Nitpick: this one change doesn't look related to the ticket.

Reply:
Removed